src/tables.ts

```diff
@@ -598,49 +598,53 @@ const accidentalColumns: Record<number, { [name: string]: number[] }> = {
+  'a+': { code: 0xe633 /*pluckedLeftHandPizzicato*/, betweenLines: false }, // Left hand pizzicato
   ao: {
-    aboveCode: 'pluckedSnapPizzicatoAbove',
-    belowCode: 'pluckedSnapPizzicatoBelow',
+    aboveCode: 0xe631 /*pluckedSnapPizzicatoAbove*/,
+    belowCode: 0xe630 /*pluckedSnapPizzicatoBelow*/,
     betweenLines: false,
   }, // Snap pizzicato
-  ah: { code: 'stringsHarmonic', betweenLines: false }, // Natural harmonic or open note
-  'a@': { aboveCode: 'fermataAbove', belowCode: 'fermataBelow', betweenLines: false }, // Fermata
-  'a@a': { code: 'fermataAbove', betweenLines: false }, // Fermata above staff
-  'a@u': { code: 'fermataBelow', betweenLines: false }, // Fermata below staff
-  'a@s': { aboveCode: 'fermataShortAbove', belowCode: 'fermataShortBelow', betweenLines: false }, // Fermata short
-  'a@as': { code: 'fermataShortAbove', betweenLines: false }, // Fermata short above staff
-  'a@us': { code: 'fermataShortBelow', betweenLines: false }, // Fermata short below staff
-  'a@l': { aboveCode: 'fermataLongAbove', belowCode: 'fermataLongBelow', betweenLines: false }, // Fermata long
-  'a@al': { code: 'fermataLongAbove', betweenLines: false }, // Fermata long above staff
-  'a@ul': { code: 'fermataLongBelow', betweenLines: false }, // Fermata long below staff
-  'a@vl': { aboveCode: 'fermataVeryLongAbove', belowCode: 'fermataVeryLongBelow', betweenLines: false }, // Fermata very long
-  'a@avl': { code: 'fermataVeryLongAbove', betweenLines: false }, // Fermata very long above staff
-  'a@uvl': { code: 'fermataVeryLongBelow', betweenLines: false }, // Fermata very long below staff
-  'a|': { code: 'stringsUpBow', betweenLines: false }, // Bow up - up stroke
-  am: { code: 'stringsDownBow', betweenLines: false }, // Bow down - down stroke
-  'a,': { code: 'pictChokeCymbal', betweenLines: false }, // Choked
+  ah: { code: 0xe614 /*stringsHarmonic*/, betweenLines: false }, // Natural harmonic or open note
+  'a@': { aboveCode: 0xe4c0 /*fermataAbove*/, belowCode: 0xe4c1 /*fermataBelow*/, betweenLines: false }, // Fermata
+  'a@a': { code: 0xe4c0 /*fermataAbove*/, betweenLines: false }, // Fermata above staff
+  'a@u': { code: 0xe4c1 /*fermataBelow*/, betweenLines: false }, // Fermata below staff
+  'a@s': { aboveCode: 0xe4c4 /*fermataShortAbove*/, belowCode: 0xe4c5 /*fermataShortBelow*/, betweenLines: false }, // Fermata short
+  'a@as': { code: 0xe4c4 /*fermataShortAbove*/, betweenLines: false }, // Fermata short above staff
+  'a@us': { code: 0xe4c5 /*fermataShortBelow*/, betweenLines: false }, // Fermata short below staff
+  'a@l': { aboveCode: 0xe4c6 /*fermataLongAbove*/, belowCode: 0xe4c7 /*fermataLongBelow*/, betweenLines: false }, // Fermata long
+  'a@al': { code: 0xe4c6 /*fermataLongAbove*/, betweenLines: false }, // Fermata long above staff
+  'a@ul': { code: 0xe4c7 /*fermataLongBelow*/, betweenLines: false }, // Fermata long below staff
+  'a@vl': {
```

Comment:
I admit I am not very familiar with this area of the codebase. 

What are these special codes like a@ul and a@uvl ?

Are they VexFlow specific shorthand for specifying Articulations? Since we are breaking backwards compatibility with V5, perhaps we can do a separate PR to improve this. The magic shorthand isn't easy to understand.

Reply:
yes, they are vexflow specific shorthands.

Reply:
Probably it would be better to use the code directly and calculate betweenLines on run time based on the glyph height.

Reply:
Does anyone use these shorthand codes? I don't see it used in our tests.

We can keep it for now I suppose (for legacy reasons) but this secret shorthand isn't that readable anyways. :-)

Reply:
They are used a lot in articulation_tests.ts

Reply:
>> Does anyone use these shorthand codes? I don't see it used in our tests.

>> We can keep it for now I suppose (for legacy reasons) but this secret shorthand isn't that readable anyways. :-)

> They are used a lot in articulation_tests.ts

They are also used for easyScore, which is why they're so cryptic.

If anyone uses 'fermataLongBelow' because that's how it was done in the test cases, will it still work? 

What happens if someone passes in a smufl code that's not an articulation - will it still render?  I don't know that we should prevent it, just curious.